src/components/Todos/index.js

```diff
@@ -26,20 +38,54 @@ function amendStateWithChildren(DOMSource) {
   };
 }
 
+// THE TODOS FUNCTION
+// This is the Todos component which is being exported below.
+// The sources that have been passed to `run` as well are immediately
+// deconstructed into the variables `DOM`, `hashchange`, `initialHash` & `storage`.
```

Comment:
I would rephrase this sentence as: "Using destructuring, we pick the sources `DOM`, `hashchange`, `initialHash` and `storage` from the sources object as argument."